package.json

```diff
@@ -225,7 +225,7 @@
 		"build:packages": "npm run build:package-types && node ./bin/packages/build.js",
 		"build:package-types": "node ./bin/packages/validate-typescript-version.js && tsc --build",
 		"build:plugin-zip": "./bin/build-plugin-zip.sh",
-		"build": "npm run build:packages && wp-scripts build",
+		"build": "cross-env NODE_ENV=production npm run build:packages && wp-scripts build",
```

Comment:
I'm not sure why we weren't doing this?

This let Babel knows that we're targeting production builds, and applies some prod-only enhancements. I guess it's because we didn't have any plugins that used this feature?

Reply:
I guess this was part of `npm run build:packages` internally no?

Reply:
`npm run build:packages` is also used by `npm run dev`, so I think no.

Reply:
We use different environments in `npm run build:packages`:
- `main` for `build` folders
- `module` for build-module` folders

I guess the current flow doesn't care whether it's a production or development env. We leave that part to webpack to decide.

As far as I can tell, the Babel preset for WordPress has a special handling only for the `test` env.